walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +271,115 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  public void optionalAttendeeWithWholeDayEvent() {
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_B)),
```

Comment:
Nit: The way the google linter would wrap this is:

```
new Event(
    "Event 1",
    TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
    Arrays.asList(PERSON_A)),
new Event(
    "Event 2",
    TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
    Arrays.asList(PERSON_B)),
```

although I don't think this is actually against the style guide.